.github/workflows/conformance-datapath.yaml

```diff
@@ -428,13 +428,13 @@ jobs:
             ./cilium-cli install ${{ steps.vars.outputs.cilium_install_defaults }}
 
             ./cilium-cli status --wait
-            kubectl -n kube-system exec -it daemonset/cilium  -- cilium status
+            kubectl -n kube-system exec daemonset/cilium -- cilium status
 
             ./cilium-cli connectivity test --datapath --collect-sysdump-on-failure \
               --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>"
             ./cilium-cli connectivity test --collect-sysdump-on-failure \
               --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>"
-            ./contrib/scripts/kind-down.sh --keep-registry
```

Comment:
Not related to your PR, but why do we need to kill Kind? The runner won't be reused by any other test run.

Reply:
No idea. Could it be a leftover from when the workflow was running connectivity tests twice?

Reply:
Most likely. We can remove it to save a few secs.